Update zh_cn.snbt

USS-Corvan opened the pull request:
Updated zh_cn.snbt based on the latest en_us.snbt

muon-rw commented:
I notice there are a lot of inconsistencies with line breaks and escaped formatting codes, though I'm not sure if these were due to changes by FTB Quests itself auto-processing lang files differently. 

Have you tested viewing all of the translated quests in game?

USS-Corvan commented:
> I notice there are a lot of inconsistencies with line breaks and escaped formatting codes, though I'm not sure if these were due to changes by FTB Quests itself auto-processing lang files differently.
> 
> Have you tested viewing all of the translated quests in game?

Yes, I have tested it in MMC5 v11. I usually keep the game running when I translate the lines, so that I could see the effects in-game. Still, there could be some error due to strange reasons. I've checked it again, and all the quests that **can be seen in the quest book** should now have no errors.

The line breaks are based on the changes of the latest en_us.snbt, which contains inconsistencies itself. I intend to unify the format, yet  aborted feeling it might be intentional. 

I do not know what you mean by "escaped formatting codes". If you mean something like

chapter.0D7FF8F037EFF568.title: " &4下界"

then I'd say all these formats are based on the original file (en_us.snbt),

muon-rw commented:
An escaped formatting code is when a "\" is prefaced by another "\" to prevent the first one from being interpreted as a formatting code itself. If it looks fine in-game, it should be fine! 

By the way, I've just updated quests with some new info, which you might want to include: https://github.com/LunaPixelStudios/Medieval-MC/commit/520718016eafeb69420315a86c37de54d6a095e9#diff-44c804f4f9424a06ee5d5b7666bb442d57c406bbea32de12837a0ba1e2cb664a

USS-Corvan commented:
Those are also from the original en_us file, I didn't dare changing all of them without knowing if they are intended (because they are working fine currently), for fear of breaking anything. About the new quests, thanks for informing me! I'll include them later today!



---- Replied Message ----
| From | ***@***.***> |
| Date | 02/03/2025 09:00 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [LunaPixelStudios/Medieval-MC] Update zh_cn.snbt (PR #862) |

An escaped formatting code is when a "\" is prefaced by another "\" to prevent the first one from being interpreted as a formatting code itself. If it looks fine in-game, it should be fine!

By the way, I've just updated quests with some new info, which you might want to include: 5207180#diff-44c804f4f9424a06ee5d5b7666bb442d57c406bbea32de12837a0ba1e2cb664a

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

USS-Corvan commented:
> An escaped formatting code is when a "\" is prefaced by another "\" to prevent the first one from being interpreted as a formatting code itself. If it looks fine in-game, it should be fine!
> 
> By the way, I've just updated quests with some new info, which you might want to include: [5207180#diff-44c804f4f9424a06ee5d5b7666bb442d57c406bbea32de12837a0ba1e2cb664a](https://github.com/LunaPixelStudios/Medieval-MC/commit/520718016eafeb69420315a86c37de54d6a095e9#diff-44c804f4f9424a06ee5d5b7666bb442d57c406bbea32de12837a0ba1e2cb664a)

I wasn't able to commit the updated file until now due to network issue. I have updated the zh_cn.snbt now.

Still there is a problem here: in this mode the zh_cn file would always be at least one version behind en_us file. Do you have any idea on how to solve that?

muon-rw commented:
You might try a service like https://app.github-file-watcher.com/ to receive a notification everytime MMC5's ftbquests en_us file changes, and if you submit a PR within ~24 hours of that I can generally make sure it's included. Updates/config pushes for MMC5 usually happen on Saturdays

muon-rw commented:
@USS-Corvan I've made some lang updates in 30fa397

USS-Corvan commented:
> @USS-Corvan I've made some lang updates in [30fa397](https://github.com/LunaPixelStudios/Medieval-MC/commit/30fa397fd0c007b71ccc55f8be8aeb5b1be99c5f)
Thank you for informing me! I'll work on it right now.
